tpl_functions.php

```diff
@@ -1447,15 +1447,20 @@ function bootstrap3_pageinfo($ret = false) {
       if (isset($INFO['editor'])) {
 
         $user = editorinfo($INFO['editor']);
+        $avatar_conf = bootstrap3_conf('userAvatar');
 
-        if (bootstrap3_conf('useGravatar')) {
+        if ($avatar_conf != 'noAvatar') {
 
           global $auth;
           $user_data = $auth->getUserData($INFO['editor']);
 
-          $gravatar_img = ml(get_gravatar($user_data['mail'], 16).'&.jpg', array('cache' => 'recache', 'w' => 16, 'h' => 16));
+          if ($avatar_conf == 'Gravatar') {
+            $avatar_img = ml(get_gravatar($user_data['mail'], 16).'&.jpg', array('cache' => 'recache', 'w' => 16, 'h' => 16));
+          }elseif ($avatar_conf == 'localAvatar') {
+            $avatar_img = '../../wiki/data/media/user/'.$_SERVER['REMOTE_USER'].'.jpg?nocache';
```

Comment:
../../wiki/[...] won't work: it works in my website since I put every DokuWiki file inside a "wiki" folder. I don't know how to automatically get the "data/media" url, so this has to be fixed!